samples/SmokeTest/app.ts

```diff
@@ -0,0 +1,41 @@
+import {KeyVaultSecrets} from "./KeyVaultTest";
+import {EventHubs} from "./EventHubsTest";
+import {BlobStorage} from "./BlobStorage";
+import {CosmosDB} from "./CosmosDB";
+
+function welcomeMessage(){
+    console.log("=============================================");
+    console.log("Smoke Test Samples for SDK Track 2 libraries");
+    console.log("=============================================");
+    console.log();
+}
+
+async function runEachSample(sample){
+    console.log("------------------------");
+    console.log(sample.service);
+    console.log("------------------------");
+    console.log(sample.description);
+
+    await sample.Run();
+
+    console.log();
+}
+
+///This function is going to call each sample to run it.
+async function main(){
+    welcomeMessage();
+
+    let keyVault = new KeyVaultSecrets();
```

Comment:
I was hoping this would simply be a call to a static method into the keyvault class. in general we want this app file to be as simple as possible which ensures that the standalone samples are self-contained.